frontend/packages/core/src/card.tsx

```diff
@@ -8,19 +8,15 @@ import {
   CardActionAreaProps,
   CardHeader as MuiCardHeader,
 } from "@material-ui/core";
+import type { SpacingProps as MuiSpacingProps } from "@material-ui/system";
 import { spacing } from "@material-ui/system";
 
-import { StyledTypography } from "./typography";
```

Comment:
in doing this cleanup https://github.com/lyft/clutch/pull/1720/commits/0258d972f274e671ca154b4b345e6df41a596104#r700359762, double checked the typography for the landing card and saw we should be using the Typography component instead.